QUEST_LV_0100.tsv

```diff
@@ -355,13 +355,13 @@ QUEST_LV_0100_20150317_000350	$I don't know why they suddenly started to prey on
 QUEST_LV_0100_20150317_000351	$Thank you.{nl}I envy your ability.
 QUEST_LV_0100_20150317_000352	$Eastern Woods Search Scout
 QUEST_LV_0100_20150317_000353	$Oh, it's you.{nl}I have heard from the Operations Officer.{nl}
-QUEST_LV_0100_20150317_000354	{memo X}$A Vubbe Fighter is hiding deep in the Nudegi Felled Area.{nl}We got the orders to kill it, but it may be safer to wait for additional troops.
+QUEST_LV_0100_20150317_000354	{memo X}$A Vubbe Fighter is hiding deep in the Nudegi Felled Area.{nl}We received orders to kill it, but it may be safer to wait for additional troops.
 QUEST_LV_0100_20150317_000355	$I'm just a Search Scout, but this was my first time seeing a Vubbe Fighter up close.
-QUEST_LV_0100_20150317_000356	$Awesome. You got rid of it by yourself.{nl}Go ahead and report to Aras. He'll be very pleased.
-QUEST_LV_0100_20150317_000357	$You mean you really defeated the Vubbe Fighter?{nl}Well then, that's a relief. Now the number of monsters will no longer increase.
+QUEST_LV_0100_20150317_000356	$Amazing, you got rid of it by yourself?{nl}Report to Aras, he'll be very pleased to hear this.
+QUEST_LV_0100_20150317_000357	$You really defeated the Vubbe Fighter alone!{nl}What a relief, now the number of monsters will stop increasing.
 QUEST_LV_0100_20150317_000358	$Of course. As promised, I will grant you access to the Miners' Village.{nl}Please wait.
 QUEST_LV_0100_20150317_000359	$If you find any clues, let the Operations Officer in the northern area know about it.
-QUEST_LV_0100_20150317_000360	$I think the battle at the Miners' Village is in a very bad situation.{nl}It's too late to move the troops so please go on ahead to support the Miners' Village.
+QUEST_LV_0100_20150317_000360	$The Miners' Village is in a dire situation.{nl}It's too late to organize the troops so please go ahead and support the Miners' Village.
```

Comment:
We've already come to agreement that Miners' Village is the name to use. Not Miner's Village.